src/event/event.rs

```diff
@@ -78,6 +78,13 @@ impl Default for Event {
 }
 
 impl Event {
+    /// Returns an `Iterator` for `Attributes`
```

Comment:
Wrap the types in [] brackets so it generates the links in the docs like that: https://docs.rs/cloudevents-sdk/0.1.0/cloudevents/event/index.html (look at `Event`):

```rust
/// Returns an [`Iterator`] for [`Attributes`]
```